build/dependencies.props

```diff
@@ -1,16 +1,25 @@
 <Project>
   <PropertyGroup>
-    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
+    <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
+    <CoreFxVersion>4.3.0</CoreFxVersion>
+    <DapperVersion>1.*</DapperVersion>
+    <InternalAspNetCoreSdkVersion>2.0.0-*</InternalAspNetCoreSdkVersion>
+    <JsonNetVersion>9.0.1</JsonNetVersion>
+    <NETStandardImplicitPackageVersion>$(BundledNETStandardPackageVersion)</NETStandardImplicitPackageVersion>
     <RuntimeFrameworkVersion>2.0.0-*</RuntimeFrameworkVersion>
```

Comment:
@natemcmaster Is this used by KoreBuild anywhere? I don't see it in other repos. Wondering if I can remove it.